beam-core/Database/Beam/Backend/SQL/Types.hs

```diff
@@ -17,3 +18,8 @@ newtype SqlSerial a = SqlSerial { unSerial :: a }
   deriving (Show, Read, Eq, Ord)
 instance FromBackendRow be x => FromBackendRow be (SqlSerial x) where
   fromBackendRow = SqlSerial <$> fromBackendRow
+instance Json.FromJSON a => Json.FromJSON (SqlSerial a) where
+  parseJSON a = SqlSerial <$> Json.parseJSON a
+instance Json.ToJSON a => Json.ToJSON (SqlSerial a) where
+  toJSON (SqlSerial a) = Json.toJSON a
+  toEncoding (SqlSerial a) = Json.toEncoding a
```

Comment:
Can you add a newline at the end of the file here?

Reply:
Yep will do.